pxl_scripts/px/redis_data/redis_data.pxl

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Redis Data Tracer
+
+This script traces all Redis data on the cluster.
+'''
+
+import px
+
+
+def redis_data(start: str, num_head: int):
+
+    # redis_events.beta is a beta table, which is subject to change.
+    df = px.DataFrame(table='redis_events.beta', start_time=start)
+    df = add_source_dest_columns(df)
+
+    # Filter source column.
+    # df = df[df.source == 'default/cartservice']
+
+    # Restrict number of results.
+    df = df.head(num_head)
+
+    df = add_source_dest_links(df, start)
+    df = df['time_', 'source', 'destination', 'remote_port', 'cmd',
+            'cmd_args', 'resp', 'latency_ns']
+
+    return df
+
+
+def add_source_dest_columns(df):
+    ''' Adds source, destination columns to the dataframe.
```

Comment:
I would either put the docstring or not, some of your funcs have it some don't. Also I think it would be more understandable if instead of a docstring, it instead explained what source and destination columns are in this context and what this logic is doing (in plain language)

Reply:
Check out the new comment.